api/bundle/models/bundle.settings.json

```diff
@@ -44,6 +44,9 @@
     },
     "ImageAltText_FR": {
       "type": "string"
+    },
+    "slug": {
+      "type": "uid"
```

Comment:
This looks accidentally added/not related to removing the collection type

Reply:
Good catch, I'll add this to a separate PR